e2e/fixtures/ssg-wildcard/src/pages/_layout.tsx

```diff
@@ -1,14 +1,5 @@
 import type { ReactNode } from 'react';
 
-const Layout = ({ children }: { children: ReactNode }) => (
-  <html>
-    <head>
-      <title>Waku</title>
```

Comment:
I think having this title is important.

Reply:
added the title back just inside the div which is then lifted up to the head. react does this for us now, right?

Reply:
Yep.